components/FixedBanner.vue

```diff
@@ -131,7 +131,7 @@ export default {
 
 <template>
   <div v-if="showBanner">
-    <div v-if="!showAsDialog" class="banner banner-banner" :style="bannerStyle" :class="{'banner-consent': consent}">
+    <div v-if="!showAsDialog" class="banner" :style="bannerStyle" :class="{'banner-consent': consent}">
```

Comment:
Your changes only remove the text ellipsis style when the banner overflows - it does not allow the banner to have extra height - you need to remove `height: 2em;` from the `banner style.